src/OSPSuite.Presentation/Presenters/AbstractExplorerPresenter.cs

```diff
@@ -380,5 +382,13 @@ public virtual void Handle(ProjectClosedEvent eventToHandle)
       {
          _view.DestroyNodes();
       }
+
+      protected void RefreshTreeAfterRename()
```

Comment:
@Yuri05 I am creating this method in core so that it can be used in MoBi and PKSim. Credit to you:)